src/main/java/com/iridium/iridiumskyblock/gui/BlockValueGUI.java

```diff
@@ -48,7 +48,13 @@ public void addContent(Inventory inventory) {
 
         if (guiType == BlockValueType.BLOCK) {
             IridiumSkyblock.getInstance().getBlockValues().blockValues.entrySet().stream()
-                    .filter(valuableBlockEntry -> valuableBlockEntry.getValue().page == page)
+                    .filter(valuableBlockEntry -> {
+                        if (this.page == 1) {
+                            return valuableBlockEntry.getValue().page <= 1;
+                        } else {
+                            return valuableBlockEntry.getValue().page == page;
+                        }
+                    })
```

Comment:
Looks cleaner in my opinion but it's up to you
```suggestion
                    .filter(valuableBlockEntry -> (valuableBlockEntry.getValue().page < 1 ? 1 :  valuableBlockEntry.getValue().page) == page)
```

Reply:
we must check the page is 1